packages/Organisms/SiteSectionNavigation/menu.vue

```diff
@@ -210,12 +211,16 @@ export default {
     }
 
     &[data-depth="1"] {
+
       #{$root}__item {
         &::before {
           left: $rpl-component-padding-s;
           right: $rpl-component-padding-s;
           background-color: $rpl-section-menu-first-level-item-background-color;
         }
+        #{$root}__item-link {
+          padding-left: $rpl-section-menu-item-left-padding + rem(1 * 16px);
```

Comment:
Hi @FleetAdmiralButter please use `$rpl-space-4` to replace `rem(1 * 16px)`. `$rpl-space-4 * 2` is same as `rem(2 * 16px)`.